eng/common/TestResources/setup-enviroments.yml

```diff
@@ -26,7 +26,7 @@ steps:
       targetType: inline
```

Comment:
This filename has a typo missing an `n`: setup-enviro**m**ents.yaml

Reply:
Thanks for catching. It should be obvious how little testing I've done so far :)